src/screens/Settings/element/ProfileSettingItem.js

```diff
@@ -39,10 +39,9 @@ const styles = StyleSheet.create({
     marginBottom: 12
   },
   textCard: {
-    fontFamily: fonts.inter[700],
-    fontWeight: '800',
+    fontFamily: fonts.inter[800],
     fontSize: 14,
-    color: COLORS.black,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The font family and color of the textCard have been changed. The fontWeight property has also been removed. Ensure these changes are in line with the design specifications and maintain good readability.
```diff
- fontFamily: fonts.inter[700],
- fontWeight: '800',
- color: COLORS.black,
+ fontFamily: fonts.inter[800],
+ color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->